doc/nrf/releases_and_maturity/releases/release-notes-3.0.0-preview1.rst

```diff
@@ -108,6 +108,14 @@ Security
     * Support for HKDF-Expand and HKDF-Extract in CRACEN.
     * Support for Ed25519ph(HashEdDSA) to CRACEN.
 
+  * Updated the Oberon PSA core to version 1.3.4 which introduces support for the following:
```

Comment:
Please update https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst instead.

Reply:
Fully agree. Once again, I forgot to check the file name.

Reply:
Sure, I moved the change to the suggested file and I integrated @peknis suggestion.